PKGBUILD

```diff
@@ -0,0 +1,20 @@
+# Maintainer: xeon-zolt <xeon.harsh@gmail.com>
+pkgname=meilix-systemlock
+pkgver=1
+pkgrel=1
+
+pkgdesc="Melix System Lock"
+arch=('i686' 'x86_64')
```

Comment:
i will add testing for this too in travis its just to create the pacman package
we use travis to test code buddy